<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page3</class>
 <widget class="QWizardPage" name="Page3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>13</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Create VDF</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>72</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Variables to include:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>68</y>
     <width>96</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Time Step</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>95</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string># of Timesteps</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numtsSpinner">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>95</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="startTimeSpinner">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>68</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="advanceOptionButton">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>332</y>
     <width>143</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string> Advanced</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectAllButton">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>96</y>
     <width>94</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newVarButton">
   <property name="geometry">
    <rect>
     <x>206</x>
     <y>96</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearAllButton">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>96</y>
     <width>79</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>126</y>
     <width>439</width>
     <height>200</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="vdfCommentButton">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>332</y>
     <width>147</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Comment</string>
   </property>
  </widget>
  <widget class="QLabel" name="createVdfLabel">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>11</y>
     <width>59</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="showCommandButton">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>332</y>
     <width>147</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Command</string>
   </property>
  </widget>
  <widget class="QLabel" name="processIndicator">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Analyzing Files...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
